modules/Microsoft.Sql/managedInstances/.test/common/dependencies.bicep

```diff
@@ -16,7 +16,6 @@ param keyVaultName string
 @description('Optional. The location to deploy resources to.')
 param location string = resourceGroup().location
 
-var sqlMiVnetAddressPrefix = '10.0.0.0/16'
```

Comment:
As mentioned when opening this PR, although I agree with the general recommendation, I suggest to keep this variable giving the strict correlation between vnet and subnet address space

Reply:
But then what is the rule? :D 

> Don't create artificial variables, unless absolutly necessary - or because of high correlation with another variable?

Reply:
"Absolutely necessary" is difficult to define, I'd go with something even more difficult to define: "Don't create artificial variables, unless it makes sense" 😆

Reply:
Love it. #needsFurtherDiscussion ;)

Reply:
resolved as per our discussion